src/views/Predictions/components/RoundCard/CalculatingCard.tsx

```diff
@@ -45,7 +46,7 @@ const CalculatingCard: React.FC<CalculatingCardProps> = ({ round }) => {
           <MultiplierArrow betPosition={BetPosition.BEAR} isDisabled />
         </CardBody>
       </Card>
-      {tooltipVisible && tooltip}
+      {tooltipVisible && ReactDOM.createPortal(tooltip, document.getElementById('root'))}
```

Comment:
Let's maybe not change prediction stuff for now, since market is closed and its hard to test if it makes it better or worse. And I remember tooltips were fine there (at least in English)

Reply:
done